frameworks/digital-outcomes-and-specialists-2/manifests/edit_brief_response.yml

```diff
@@ -1,11 +1,10 @@
 -
-  name: Apply for this opportunity
+  name: Apply for an opportunity
```

Comment:
Was a change carried over from when we made this new supplier flow manifest - https://github.com/alphagov/digitalmarketplace-frameworks/commit/47e4b545d75a4df23920f5af291a51b0c62b9df2


We don't actually show the name of the section anywhere for the new supplier flow so it shouldn't matter what the name is but maybe I'll change them both to be 'this' for consistency.